package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@nextcloud/vue",
-  "version": "8.18.0",
+  "version": "8.18.1",
```

```suggestion
  "version": "8.19.0",
```

Semantically new props are feature

---

Alright :blush: